src/codegen/spirv/spec.zig

```diff
@@ -1302,7 +1302,7 @@ pub const Decoration = enum(u32) {
     RegisterINTEL = 5825,
     MemoryINTEL = 5826,
     NumbanksINTEL = 5827,
-    BankwidthINTEL = 5828,
```

Comment:
?

Reply:
`BankwidthINTEL` is correct https://www.khronos.org/registry/SPIR-V/specs/unified1/SPIRV.html.